config/default/kustomization.yaml

```diff
@@ -14,9 +14,6 @@ namePrefix: gatekeeper-operator-
 commonLabels:
```

Comment:
Do we need to commit these changes?

Reply:
I can investigate the changes but I'd like to see `make release` and `make bundle` idempotent.

Reply:
:100: agree. In the past I've had issues trying to maintain what the correct idempotent state of the file should be, as certain commands would alter it again and then break CI because it detected a change that was not committed. This may have been fixed now as clearly the tests have passed.

Reply:
Can you run `make release` and see if the results match? It's kustomize 
```
$ kustomize version
{Version:unknown GitCommit:$Format:%H$ BuildDate:1970-01-01T00:00:00Z GoOs:linux GoArch:amd64}
```